x-pack/plugins/fleet/server/routes/epm/index.ts

```diff
@@ -237,7 +238,7 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
       path: EPM_API_ROUTES.INSTALL_BY_UPLOAD_PATTERN,
       options: {
         body: {
-          accepts: ['application/gzip', 'application/zip'],
+          accepts: ['application/gzip', 'application/zip'], // TODO: openapi missing
```

Comment:
generated OAS, content type is not correct:

```
GET kbn:/api/oas?pathStartsWith=/api/fleet/epm/packages

 "paths": {
    "/api/fleet/epm/packages": {
...
 "post": {
        "requestBody": {
          "content": {
            "application/json; Elastic-Api-Version=2023-10-31": {
              "schema": {
                "format": "binary",
                "type": "string"
              }
            }
          }
        },
```

Reply:
Good catch, this is a known limitation and should be relatively straight forward to fix!